lib/parse-args.js

```diff
@@ -5,9 +5,11 @@ const { readFileSync } = require('fs')
 const Yargs = require('yargs/yargs')
 const { applyExtends } = require('yargs/helpers')
 const parser = require('yargs-parser')
-const { resolve } = require('path')
+const { resolve, isAbsolute } = require('path')
 
 function buildYargs (withCommands = false) {
+  let defaultReportDir = resolve(process.cwd(), 'coverage')
```

Comment:
Function level variable defaultReportDir that is used in the reports-directory and tmp-directory arguments.